### ISSUE TITLE
Redraw datatable [Question]

### ISSUE THREAD

**Issue by justpoypoy** [jeroennoten/Laravel-AdminLTE#1209]
### Describe the bug
Hello, I am using datatable clientside, When i am save button on modal, data on datatable not refresh after success save button using ajax.
```$('#save_button').click(function(e) {
                $.ajax({
                    data: {{ dataRequest }},
                    url: {{ URL }},
                    type: "POST",
                    dataType: 'json',
                    success: function(data) {
                        $('#showModalCreate').modal('hide');
                        $('#table1').DataTable().draw();
                    },
                    error: function(data) {
                        $('#save_button').html('Save Changes');
                    }
                });
        });```

### Expected behavior
refresh datatable after modal hide and get update row data


| Item    | Version
| ------- | -------
| Laravel | 8.75
| Project | Private
| OS      | Alpine Linux v3.13 (Docker)

**Comment by justpoypoy** [comment]
`
@php
$heads = [
    'ID',
    'Name',
    ['label' => 'Phone', 'width' => 40],
    ['label' => 'Actions', 'no-export' => true, 'width' => 5],
];

$btnEdit = '<button class="btn btn-xs btn-default text-primary mx-1 shadow" title="Edit">
                <i class="fa fa-lg fa-fw fa-pen"></i>
            </button>';
$btnDelete = '<button class="btn btn-xs btn-default text-danger mx-1 shadow" title="Delete">
                  <i class="fa fa-lg fa-fw fa-trash"></i>
              </button>';
$btnDetails = '<button class="btn btn-xs btn-default text-teal mx-1 shadow" title="Details">
                   <i class="fa fa-lg fa-fw fa-eye"></i>
               </button>';

$config = [
    'data' => [
        [22, 'John Bender', '+02 (123) 123456789', '<nobr>'.$btnEdit.$btnDelete.$btnDetails.'</nobr>'],
        [19, 'Sophia Clemens', '+99 (987) 987654321', '<nobr>'.$btnEdit.$btnDelete.$btnDetails.'</nobr>'],
        [3, 'Peter Sousa', '+69 (555) 12367345243', '<nobr>'.$btnEdit.$btnDelete.$btnDetails.'</nobr>'],
    ],
    'order' => [[1, 'asc']],
    'columns' => [null, null, null, ['orderable' => false]],
];
@endphp

{{-- Minimal example / fill data using the component slot --}}
<x-adminlte-datatable id="table1" :heads="$heads">
    @foreach($config['data'] as $row)
        <tr>
            @foreach($row as $cell)
                <td>{!! $cell !!}</td>
            @endforeach
        </tr>
    @endforeach
</x-adminlte-datatable>`

I am using your example on wiki...

**Comment by dfsmania** [comment]
But that's just an example with **hard coded** data to show how the table will render. I'm not sure how you expect the `AJAX` request you share at the first post to change this **hard coded** dataset. A real scenario will use a database on the server-side, and your `datatable` will be usually filled with some of that data, then the `AJAX` request will change the data on that database, and finally after this, the idea is to reload your page in order to show the new data (usually a redirection to the same page is triggered from the controller you hit with the `AJAX` request to emulate an auto-reload).

I would love to create an example with code to show you the idea, but I don't have the time. So I hope you can find your way into solving your requirements.
